android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -25,7 +25,7 @@
  * Created by vyakunin on 3/18/18.
  */
 
-public class HistoryTextFragment extends Fragment implements MoodListener {
+public class HistoryTextFragment extends Fragment implements MoodListener, AdapterView.OnItemLongClickListener {
```

Comment:
Can you move "implements MoodListener, AdapterView.OnItemLongClickListener" one line down please?
It's better for readability.

Reply:
Yeah.